src/dialects/postgres/query-generator-typescript.ts

```diff
@@ -48,4 +51,34 @@ export class PostgresQueryGeneratorTypeScript extends AbstractQueryGenerator {
       'GROUP BY i.relname, ix.indexrelid, ix.indisprimary, ix.indisunique, ix.indkey ORDER BY i.relname;',
     ]);
   }
+
+  removeIndexQuery(tableName: TableNameOrModel, indexNameOrAttributes: string | string[], options: RemoveIndexQueryOptions) {
+    if (options) {
+      rejectInvalidOptions(
+        'removeIndexQuery',
+        this.dialect.name,
+        REMOVE_INDEX_QUERY_SUPPORTABLE_OPTIONS,
+        new Set(['concurrently', 'ifExists', 'cascade']),
```

Comment:
```suggestion
        new Set<string>(['concurrently', 'ifExists', 'cascade']),
```
To be in line with the other dialects, but I assume that TS would already know that the set is made of strings so I'm not sure if we even need it